extend/publish/checklist.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Checklist
+permalink: /extend/publish/checklist/
+---
+
+* TOC
+{:toc}
+
+This checklist is used for the last check of the component before sending the publish request.
+See [Publish Component tutorial](/extend/publish/) for details.
+
+**Developer Portal**
+- <input type="checkbox"> Component's name doesn't contain words like `extractor`, `application`, and `writer`.
+- <input type="checkbox"> Component icons is representative and has reasonable quality.
+- <input type="checkbox"> The short description describes the **service**, NOT `This extractor extracts ...`.
+- <input type="checkbox"> Licensing information is valid, and the vendor description is current.
+- <input type="checkbox"> License, Documentation and Source code URLs are publicly accessible, no link to a private repository.
+- <input type="checkbox"> The tag is set to the expected value and uses [semantic versioning](https://semver.org/).
+- <input type="checkbox"> The correct [data flow is set in UI options](/extend/publish/#component-name-and-description).
+
+
+**Component Configuration**
+- <input type="checkbox"> Sensitive values [use encryption](/overview/encryption/).
+- Configuration description (if used)
+  - <input type="checkbox"> Contains only 3 `###` and level 4 `####` headers.
+  - <input type="checkbox"> Doesn't describe all properties. Property `description` should be used instead if possible. 

```

Comment:
tomhule nerozumím

Reply:
Tak Leo rad napise do `Configuration description` vsetky polozky s popisom co tam treba zadat, ... ale do formulara ich bud neda alebo da duplicitne. A teda musis pozriet do `Configuration description` aby si vedel co zadat ... alebo mas dlhu configuration description, ktora nenesie ziadnu pridanu hodnotu.

Ak to dava zmysel, tak to mozem skusit opisat nejak podrobnejsie.

Reply:
Aha, chápu. Co kdyby se přehodily odrážky "Configuration description" A "Configuration Row & Schema". Pak by se dalo napsat "Doesn't repeat what it is obvious from configuration schema"

Reply:
:+1: super napad, opravene https://github.com/keboola/developers-docs/pull/168/commits/37561c85d6f7d5e0b918ee6fac623a7cd576affd